whc-shared/src/main/java/com/ibm/whc/deid/shared/pojo/config/masking/NumberVarianceMaskingProviderConfig.java

```diff
@@ -84,6 +85,24 @@ public void setMaskLimitDown(double maskLimitDown) {
     this.maskLimitDown = maskLimitDown;
   }
 
+  @Override
+  public void validate() throws InvalidMaskingConfigurationException {
+    if (augmentLowerBound < 0.0) {
+      throw new InvalidMaskingConfigurationException("`augmentLowerBound` must be greater than 0");
+    }
+    if (augmentUpperBound < 0.0) {
+      throw new InvalidMaskingConfigurationException("`augmentUpperBound` must be greater than 0");
+    }
+    if (augmentMask && augmentLowerBound > augmentUpperBound) {
+      throw new InvalidMaskingConfigurationException(
+          "`augmentLowerBound` must be less than `augmentUpperBound` when `augmentMask` is true");
+    }
+    if (augmentMask && augmentLowerBound == augmentUpperBound && !resultWithPrecision) {
+      throw new InvalidMaskingConfigurationException(
+          "`augmentUpperBound` must not be equal to `augmentLowerBound` when `augmentMask` is true");

```

Comment:
maybe say `resultWithPrecision` is false instead of mentioning augmentMask since equal is OK when resultWithPrecision

Reply:
done